docs_src/sidebar.json

```diff
@@ -1,7 +1,8 @@
 {
     "Getting started": {
         "Overview": "/",
-        "Quick Start": "/quick_start.html"
+        "Quick Start": "/quick_start.html",
+        "Performance Tips and Tricks": "/dev/performance.html"
```

Comment:
Please move this to the advanced tutorials sections.

Reply:
I've updated the branch but it still mentions changes required so I'm going to resolve this conversation in the hope that cleans it up. Apologies if I've just added more notifications unnecessarily.